src/main/java/sparksoniq/exceptions/codes/ErrorCodes.java

```diff
@@ -22,6 +22,8 @@
 
 public class ErrorCodes {
 
+    public static final String ErrorRetrievingResourceCode = "FODC0002";
```

Comment:
Feels nice to see the alphabetical order in action:) 

I just wanted to point out that there is also the docs/Error codes.md file which I recently updated to match the Errorcodes.java. I don't know if it serves any other purpose than being a readme file for that part of the repo. Feel free to add this new exception there too if you feel like it.

Reply:
Thanks, @CanBerker very good point. I extended the documentation list and also documented the function.